styx-common/src/main/java/com/spotify/styx/state/RunState.java

```diff
@@ -184,6 +184,22 @@ public RunState created(WorkflowInstance workflowInstance, String executionId, S
     }
 
     @Override
+    public RunState info(WorkflowInstance workflowInstance, String message) {
```

Comment:
It seems automatter and inner enums doesn't play nice with adt4j, try adding `StateData.MessageLevel` as another parameter or replacing `String message` with `StateData.Message message` and you will get the following error.

```
Error:java: java.lang.IllegalStateException: Inner class should always be defined if outer class is defined: inner class com.spotify.styx.state.StateData.MessageLevel, enclosing class com.helger.jcodemodel.JDefinedClass(com.spotify.styx.state.StateData):
  java.lang.IllegalStateException: Inner class should always be defined if outer class is defined: inner class com.spotify.styx.state.StateData.MessageLevel, enclosing class com.helger.jcodemodel.JDefinedClass(com.spotify.styx.state.StateData)
  	at com.helger.jcodemodel.meta.DecidedErrorTypesModelsAdapter.getClass(DecidedErrorTypesModelsAdapter.java:177)
  	at com.helger.jcodemodel.meta.DecidedErrorTypesModelsAdapter.ref(DecidedErrorTypesModelsAdapter.java:289)
  	at com.helger.jcodemodel.meta.TypeMirrorToJTypeVisitor.visitDeclared(TypeMirrorToJTypeVisitor.java:143)
  	at com.helger.jcodemodel.meta.TypeMirrorToJTypeVisitor.visitDeclared(TypeMirrorToJTypeVisitor.java:68)
  	at com.sun.tools.javac.code.Type$ClassType.accept(Type.java:944)
  	at com.helger.jcodemodel.meta.DecidedErrorTypesModelsAdapter.toJType(DecidedErrorTypesModelsAdapter.java:298)
  	at com.helger.jcodemodel.meta.ClassFiller.fillClass(ClassFiller.java:131)
  	at com.helger.jcodemodel.meta.DecidedErrorTypesModelsAdapter.defineInnerClass(DecidedErrorTypesModelsAdapter.java:270)
  	at com.helger.jcodemodel.meta.ClassFiller.fillClass(ClassFiller.java:110)
  	at com.helger.jcodemodel.meta.DecidedErrorTypesModelsAdapter.defineTopLevelClass(DecidedErrorTypesModelsAdapter.java:227)
  	at com.helger.jcodemodel.meta.DecidedErrorTypesModelsAdapter.defineClass(DecidedErrorTypesModelsAdapter.java:191)
  	at com.helger.jcodemodel.meta.DecidedErrorTypesModelsAdapter.getClass(DecidedErrorTypesModelsAdapter.java:161)
  	at com.helger.jcodemodel.meta.DecidedErrorTypesModelsAdapter.getClass(DecidedErrorTypesModelsAdapter.java:168)
  	at com.helger.jcodemodel.meta.DecidedErrorTypesModelsAdapter.ref(DecidedErrorTypesModelsAdapter.java:289)
  	at com.helger.jcodemodel.meta.TypeMirrorToJTypeVisitor.visitDeclared(TypeMirrorToJTypeVisitor.java:143)
  	at com.helger.jcodemodel.meta.TypeMirrorToJTypeVisitor.visitDeclared(TypeMirrorToJTypeVisitor.java:68)
  	at com.sun.tools.javac.code.Type$ClassType.accept(Type.java:944)
  	at com.helger.jcodemodel.meta.DecidedErrorTypesModelsAdapter.toJType(DecidedErrorTypesModelsAdapter.java:298)
  	at com.helger.jcodemodel.meta.ClassFiller.fillClass(ClassFiller.java:143)
  	at com.helger.jcodemodel.meta.DecidedErrorTypesModelsAdapter.defineTopLevelClass(DecidedErrorTypesModelsAdapter.java:227)
  	at com.helger.jcodemodel.meta.DecidedErrorTypesModelsAdapter.defineClass(DecidedErrorTypesModelsAdapter.java:191)
  	at com.helger.jcodemodel.meta.DecidedErrorTypesModelsAdapter.getClass(DecidedErrorTypesModelsAdapter.java:161)
  	at com.helger.jcodemodel.meta.JCodeModelJavaxLangModelAdapter.getClass(JCodeModelJavaxLangModelAdapter.java:145)
  	at com.helger.jcodemodel.meta.JCodeModelJavaxLangModelAdapter.getClassWithErrorTypes(JCodeModelJavaxLangModelAdapter.java:109)
  	at com.github.sviperll.adt4j.GenerateValueClassForVisitorProcessor$ElementProcessor.processStage0(GenerateValueClassForVisitorProcessor.java:202)
  	at com.github.sviperll.adt4j.GenerateValueClassForVisitorProcessor$ElementProcessor.generateClassesWithErrors(GenerateValueClassForVisitorProcessor.java:127)
  	at com.github.sviperll.adt4j.GenerateValueClassForVisitorProcessor$ElementProcessor.generateClassesWithoutErrors(GenerateValueClassForVisitorProcessor.java:139)
  	at com.github.sviperll.adt4j.GenerateValueClassForVisitorProcessor.process(GenerateValueClassForVisitorProcessor.java:109)
  	at com.sun.tools.javac.processing.JavacProcessingEnvironment.callProcessor(JavacProcessingEnvironment.java:794)
  	at com.sun.tools.javac.processing.JavacProcessingEnvironment.discoverAndRunProcs(JavacProcessingEnvironment.java:705)
  	at com.sun.tools.javac.processing.JavacProcessingEnvironment.access$1800(JavacProcessingEnvironment.java:91)
  	at com.sun.tools.javac.processing.JavacProcessingEnvironment$Round.run(JavacProcessingEnvironment.java:1035)
  	at com.sun.tools.javac.processing.JavacProcessingEnvironment.doProcessing(JavacProcessingEnvironment.java:1176)
  	at com.sun.tools.javac.main.JavaCompiler.processAnnotations(JavaCompiler.java:1170)
  	at com.sun.tools.javac.main.JavaCompiler.compile(JavaCompiler.java:856)
  	at com.sun.tools.javac.main.Main.compile(Main.java:523)
  	at com.sun.tools.javac.api.JavacTaskImpl.doCall(JavacTaskImpl.java:129)
  	at com.sun.tools.javac.api.JavacTaskImpl.call(JavacTaskImpl.java:138)
  	at org.jetbrains.jps.javac.JavacMain.compile(JavacMain.java:180)
  	at org.jetbrains.jps.incremental.java.JavaBuilder.compileJava(JavaBuilder.java:463)
  	at org.jetbrains.jps.incremental.java.JavaBuilder.compile(JavaBuilder.java:319)
  	at org.jetbrains.jps.incremental.java.JavaBuilder.doBuild(JavaBuilder.java:231)
  	at org.jetbrains.jps.incremental.java.JavaBuilder.build(JavaBuilder.java:193)
  	at org.jetbrains.jps.incremental.IncProjectBuilder.runModuleLevelBuilders(IncProjectBuilder.java:1282)
  	at org.jetbrains.jps.incremental.IncProjectBuilder.runBuildersForChunk(IncProjectBuilder.java:956)
  	at org.jetbrains.jps.incremental.IncProjectBuilder.buildTargetsChunk(IncProjectBuilder.java:1028)
  	at org.jetbrains.jps.incremental.IncProjectBuilder.buildChunkIfAffected(IncProjectBuilder.java:915)
  	at org.jetbrains.jps.incremental.IncProjectBuilder.buildChunks(IncProjectBuilder.java:741)
  	at org.jetbrains.jps.incremental.IncProjectBuilder.runBuild(IncProjectBuilder.java:387)
  	at org.jetbrains.jps.incremental.IncProjectBuilder.build(IncProjectBuilder.java:195)
  	at org.jetbrains.jps.cmdline.BuildRunner.runBuild(BuildRunner.java:138)
  	at org.jetbrains.jps.cmdline.BuildSession.runBuild(BuildSession.java:295)
  	at org.jetbrains.jps.cmdline.BuildSession.run(BuildSession.java:125)
  	at org.jetbrains.jps.cmdline.BuildMain$MyMessageHandler$1.run(BuildMain.java:238)
  	at org.jetbrains.jps.service.impl.SharedThreadPoolImpl$1.run(SharedThreadPoolImpl.java:44)
  	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
  	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
  	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
  	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
  	at java.lang.Thread.run(Thread.java:745)
```